tests/data/policy/agent_extension_policy.rego

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
should this be under the normal Agent license instead?

Reply:
updated to normal agent license since this will be packaged with agent